soweego/linker/workflow.py

```diff
@@ -151,6 +152,10 @@ def extract_features(candidate_pairs: pd.MultiIndex, wikidata: pd.DataFrame, tar
     # Feture 4: cosine similarity on descriptions
     compare.add(StringList(constants.DESCRIPTION, constants.DESCRIPTION,
                            algorithm='cosine', analyzer='soweego', label='description_cosine'))
+
+    compare.add(DateCompare(constants.DATE_OF_BIRTH,
```

Comment:
Please also add a feature for dates of death

Reply:
Good catch. Date of death has been added as a feature in 841de5b08f0cb39a72df60693a6229a13aeb8cea